Orange/widgets/utils/itemmodels.py

```diff
@@ -869,8 +869,19 @@ class DomainModel(VariableListModel):
                  ATTRIBUTES)
     PRIMITIVE = (DiscreteVariable, ContinuousVariable)
 
-    def __init__(self, order=SEPARATED, placeholder=None,
+    def __init__(self, order=SEPARATED, separators=True, placeholder=None,
                  valid_types=None, alphabetical=False, skip_hidden_vars=True, **kwargs):
+        """
+
+        :param order: How attributes, metas and classes are ordered and where between
+                      them are separators.
+        :param separators: Separators are ignored when set to False despite being set in order.
```

Comment:
Separators are ignored -> Ignore separators.
When set to False -> When what if set to False?

I tried to reformulate the sentence, but couldn't. I believe it is because I disagree with the commit as a whole.